dnf/yum/rpmtrans.py

```diff
@@ -548,7 +550,12 @@ def _scriptError(self, amount, total, key):
         scriptlet_name = rpm.tagnames.get(amount, "<unknown>")
 
         pkg, _, _ = self._extract_cbkey(key)
-        name = pkg.name
+        if pkg is not None:
+            name = pkg.name
+        elif dnf.util.is_string_type(key):
```

Comment:
@ignatenkobrain ``key`` can be dnf.transaction.TransactionItem instance or a string. Therefore it is reachable.
``pkg`` can be None or dnf.pkg object. Therefore I really don't understand your point.